.github/workflows/default_on_push.yml

```diff
@@ -14,6 +14,9 @@ on:
     tags:
       - '*.*.*'
 
+permissions:
+  contents: read
```

Comment:
We do not need the `write` permission here, right? We're pushing to GHCR, and I haven't read whether this requires some sort of extra-permission.

Reply:
Reading content is different from writing content. You probably need `packages:write` https://github.com/ossf/scorecard/blob/6ed6c9b70e837db3e94d91eb09c45fcba4bb6b15/.github/workflows/publishimage.yml#L20

But I would start with the least and then try it to before adding additional permissions.

Reply:
Alright. Feature-freeze is over for `v11.0.0`. I'd like you to add the `packages:write` permission here :) Please also have a look at the comment by @polarathene (https://github.com/docker-mailserver/docker-mailserver/pull/2555#discussion_r853640460) so we can go ahead and merge this PR - it's a very nice addition! :D